src/utils.rs

```diff
@@ -166,8 +213,29 @@ impl Console {
                     )
                 })?;
             }
+
+            // Account for the read/write/epoll_wait elapsed time
+            if disconnect_timeout_sec.is_some() {
+                epoll_timeout_us -= start_epoll_time.elapsed().as_micros() as i128;
+            }
         }
 
+        // Clear console read fd from epoll
+        epoll::epoll_ctl(epoll_fd, EpollOp::EpollCtlDel, self.fd, None).map_err(|e| {
```

Comment:
This function will never be called if the ```read()``` or ```write()``` functions inside the loop fail. While this isn't critical at the moment, since the failure will immediately lead to the termination of the nitro-cli, we should ensure that we remove the epoll descriptor no matter what.